packages/neuron-ui/src/components/SendFieldset/index.tsx

```diff
@@ -5,6 +5,7 @@ import { ckbCore } from 'services/chain'
 
 import TextField from 'widgets/TextField'
 import Button from 'widgets/Button'
+import { MessageBar, MessageBarType } from 'office-ui-fabric-react'
```

Comment:
We'll remove `official ui fabric react` in the future, so better not to use its components here.